### PR TITLE
app-benchmarks/sysbench: bump

### PR DESCRIPTION
**Pull request by hydrapolic**

### CODE REVIEW

File: dev-libs/concurrencykit/concurrencykit-0.5.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+HOMEPAGE="http://concurrencykit.org"
+SRC_URI="http://concurrencykit.org/releases/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0 BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
```
Comment (gktrk) [inline dev-libs/concurrencykit/concurrencykit-0.5.2.ebuild]: you can just prune `IUSE`, `DEPEND`, `RDEPEND`

**Comment by hydrapolic** [comment]
Thanks for the review.

I've used vcs-snapshot, but that just means adding 1 line and remove 1 other :)

**Comment by gktrk** [comment]
Please use repoman whenever possible, the commit message `app-benchmarks: bump to 1.0_pre20170122` does not contain the package name ;) I'll fix it on my end for this one, but for the future let's pay more attention.

**Comment by hydrapolic** [comment]
Sorry, added the PN to git commits. Normally I use repoman, but I don't have the keys everywhere with me :-/

**Comment by gktrk** [comment]
This sounds strange but I think this package depends on vim:
```
configure: error: "xxd is required to build sysbench (usually comes with the vim package)"

!!! Please attach the following file when seeking support:
!!! /var/tmp/portage/app-benchmarks/sysbench-1.0_pre20170122/work/sysbench-1.0_pre20170122/config.log
 * ERROR: app-benchmarks/sysbench-1.0_pre20170122::gentoo failed (configure phase):
 *   econf failed
 * 
 * Call stack:
 *               ebuild.sh, line 115:  Called src_configure
 *             environment, line 342:  Called econf '--enable-aio' '--without-mysql' '--without-pgsql' '--without-attachsql' '--without-drizzle' '--without-oracle' '--with-system-luajit' '--with-system-ck'
 *        phase-helpers.sh, line 662:  Called __helpers_die 'econf failed'
 *   isolated-functions.sh, line 117:  Called die
 * The specific snippet of code:
 *   		die "$@"
```

installing vim fixes the problem

**Comment by gktrk** [comment]
Seems like you've also fixed up the commit for the live ebuild to the bump, was that intentional?

**Comment by hydrapolic** [comment]
Thanks, added vim-core to DEPEND, split bump/sync commit msg.

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/compare/f4a8dab...226bb3a. Thanks!

**Comment by hydrapolic** [comment]
Thank you @gktrk

**Pull request closed without merging**